java/api/src/main/java/io/ray/api/events/ActorState.java

```diff
@@ -0,0 +1,44 @@
+package io.ray.api.events;
```

Comment:
How can I understand the "events"? Would it be better to rename it to "state"? Do we need to also put it into runtimecontext?

Reply:
ActorState is primarily used for monitoring changes in the state of an actor, which is why it is placed in the "events" package.